sklearn/datasets/base.py

```diff
@@ -255,14 +255,25 @@ def load_iris():
 
     Read more in the :ref:`User Guide <datasets>`.
 
+    Parameters
+     ----------
+    return_X_y : boolean, default=False.
+        If True, returns (data, target) instead of a Bunch object.
+        See below for more information about the `data` and `target` object.
+
     Returns
     -------
+    if return_X_y == false
     data : Bunch
         Dictionary-like object, the interesting attributes are:
         'data', the data to learn, 'target', the classification labels,
         'target_names', the meaning of the labels, 'feature_names', the
         meaning of the features, and 'DESCR', the
         full description of the dataset.
+    
+    if return_X_y == true
```

Comment:
I'm afraid page source is not helping.
![image](https://cloud.githubusercontent.com/assets/12952080/16995161/77a09918-4ec8-11e6-9cf7-e4a29becaaf7.png)

Clicking on the source in line with the method signature points to https://github.com/scikit-learn/scikit-learn/blob/3f9494c/sklearn/datasets/base.py#L242
Which, I suppose, is the most recent commit I made. 

![image](https://cloud.githubusercontent.com/assets/12952080/16995267/074c831a-4ec9-11e6-9a29-5db80f3b7361.png)